src/bin/taker.rs

```diff
@@ -105,6 +105,17 @@ enum Commands {
     },
     /// Recover from all failed swaps
     Recover,
+
+    /// Backup the wallet
+    WalletBackup {
+        #[clap(long, short = 'e')]
+        encrypt: bool,
```

Comment:
Docs required for this. These docs are displayed in the help message in the cli.